src/CommonITILObject.php

```diff
@@ -7770,6 +7770,36 @@ protected function updateActors()
        // reload actors
         $this->loadActors();
 
+        // append additional actors (from glpi rules)
```

Comment:
I guess this whole block should be moved before the starting if, as it can fill the `$this->input['_actors']` array.